package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kontist/mock-solaris",
-  "version": "1.0.87",
+  "version": "1.0.88-b",
```

Comment:
Is it some kind of experimental convention?

I think in case of enum extension we can simply go with standard patch version bump?

Reply:
I needed this to experiment/run tests on the backend (with ease)
I've reverted it and we can release it after merge